docs/ops/detection/ExperimentalDetectronDetectionOutput_6.md

```diff
@@ -0,0 +1,161 @@
+## ExperimentalDetectronDetectionOutput <a name="ExperimentalDetectronDetectionOutput"></a> {#openvino_docs_ops_detection_ExperimentalDetectronDetectionOutput_6}
+
+**Versioned name**: *ExperimentalDetectronDetectionOutput-6*
+
+**Category**: Object detection
+
+**Short description**: An operation *ExperimentalDetectronDetectionOutput* performs non-maximum suppression to generate 
+the detection output using information on location and score predictions.
+
+**Detailed description**: Operation doing next steps:
+
+1.  Applies deltas to boxes sizes;
+2.  If *class_agnostic_box_regression* is `true` then operation removes predictions for background classes;
+3.  Clips boxes to image;
+4.  Applies *score_threshold* on detection scores;
+5.  Applies non-maximum suppression class-wise with *nms_threshold*;
+6.  Operation returns *max_detections_per_image* detections if total number of detections is more than it, otherwise 
+returns total number of detections and the output tensor is filled with undefined values for rest output tensor 
+elements.
+
+**Attributes**:
+
+* *score_threshold*
+
+    * **Description**: *score_threshold* attribute specifies threshold to consider only detections whose score are 
+    larger than a threshold.
+    * **Range of values**: non-negative floating point number
+    * **Type**: float
+    * **Default value**: None
+    * **Required**: *yes*
+
+* *nms_threshold*
+
+    * **Description**: *nms_threshold* attribute specifies threshold to be used in the NMS stage.
+    * **Range of values**: non-negative floating point number
+    * **Type**: float
+    * **Default value**: None
+    * **Required**: *yes*
+
+* *num_classes*
+
+    * **Description**: *num_classes* attribute specifies number of detected classes.
+    * **Range of values**: non-negative integer number
+    * **Type**: int
+    * **Default value**: None
+    * **Required**: *yes*
+
+* *post_nms_count*
+
+    * **Description**: *sampling_ratio* attribute specifies the maximal number of detections per class.
+    * **Range of values**: non-negative integer number
+    * **Type**: int
+    * **Default value**: None
+    * **Required**: *yes*
+
+* *max_detections_per_image*
+
+    * **Description**: *max_detections_per_image* attribute specifies maximal number of detections per image.
+    * **Range of values**: non-negative integer number
+    * **Type**: int
+    * **Default value**: None
+    * **Required**: *yes*
+
+* *class_agnostic_box_regression*
+
+    * **Description**: *class_agnostic_box_regression* attribute ia a flag specifies whether to delete background 
+    classes or not.
+    * **Range of values**:
+      * `true` means background classes should be deleted
+      * `false` means background classes shouldn't be deleted
+    * **Type**: boolean
+    * **Default value**: false
+    * **Required**: *no*
+
+* *max_delta_log_wh*
```

Comment:
It seems to combine two attributes. Let us decouple them

Reply:
No, this is one attribute according to source code: https://github.com/openvinotoolkit/openvino/blob/83e5bde4ea9ee7d91b852ecc9707b78b01fcc7f1/ngraph/core/include/ngraph/op/experimental_detectron_detection_output.hpp#L47

And we decide to not update exist operations, just add specifications to them as is.